.github/workflows/test.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
                 php-version:
                     - '7.3'
                     - '7.4'
+                    - '8.0'
```

Comment:
Can't we add php8 on dev-kit ?

Reply:
Yep, we can add it already, not sure why wasn't there.

Reply:
see: https://github.com/sonata-project/dev-kit/pull/1674